data/events/2021-kyiv.yml

```
@@ -0,0 +1,98 @@
+name: "2021-kyiv" # The name of the event. Four digit year with the city name in lower-case, with no spaces.
+year: "2021" # The year of the event. Make sure it is in quotes.
+city: "Kyiv" # The displayed city name of the event. Capitalize it.
+event_group: "Kyiv"
+event_twitter: "days_dev" # Change this to the twitter handle for your event such as devopsdayschi or devopsdaysmsp
+description: "Devopsdays is coming to Kyiv!" # Edit this to suit your preferences
+ga_tracking_id: "" # If you have your own Google Analytics tracking ID, enter it here. Example: "UA-74738648-1"
+
+# All dates are in unquoted 2021-MM-DDTHH:MM:SS+TZ:TZ, like this:
+#   variable: 2019-01-04T00:00:00+02:00
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2021-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate: 2021-04-22T00:00:00+03:00 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: 2021-04-24T23:59:59+03:00 # The end date of your event. Leave blank if you don't have a venue reserved yet.
+
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start: 2021-01-01T00:00:00+03:00 # start accepting talk proposals.
+cfp_date_end: 2021-03-15T23:59:59+03:00 # close your call for proposals.
+cfp_date_announce:  # inform proposers of status
+
+cfp_open: "true"
+cfp_link: "https://forms.gle/DR5omkuu8NfG4RCm9" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
+
+registration_date_start: 2021-01-01T00:00:00+03:00 # start accepting registration. Leave blank if registration is not open yet
+registration_date_end: 2021-04-23T23:59:59+03:00 # close registration. Leave blank if registration is not open yet.
+
+registration_closed: "" #set this to true if you need to manually close registration before your registration end date
+registration_link: "https://636496.typeform.com/to/EPjxtL" # If you have a custom registration link, enter it here. This will control the Registration menu item as well as the "Register" button.
+
+# Location
+#
+coordinates: "50.449379,30.5209386" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+location: "Kyiv" # Defaults to city, but you can make it the venue name.
+#
+location_address: "Online" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+ # - name: propose
+ # - name: location
```

```suggestion
nav_elements: # List of pages you want to show up in the navigation of your page.
  - name: propose
    url: https://forms.gle/DR5omkuu8NfG4RCm9
 # - name: location
```
Optional, but additional ways to encourage proposals are never a bad thing imho. :)